katana/katana.py

```diff
@@ -90,6 +93,227 @@ def readconfig():
+            object_list_new.append(object_list)
+            class_list = (object_list.split('=')[0]).strip()
+        Arguments = values.find('Arguments')
+        # Finding all the arguments and it's values for
+        # each sub-keyword
+        if Arguments is not None:
+            argument_list = Arguments.findall('argument')
+            if len(argument_list) == 1:
+                for argument in argument_list:
+                    argument_name = argument.get('name')
+                    if argument_name is not None:
+                        arg_value = argument.get('value')
+                        if arg_value is None or arg_value is False:
+                            arg_value = argument.text
+                        arg_name = argument_name + "=" + "\"" + arg_value + "\""
+                        kw_name_1.append("{0}.{1}({2})".format(class_list, kw_name, arg_name))
+                        # Forming doc string for wrapper keyword by using sub keyword details
+                        doc_string_value.append(string .format(kw_name, driver, arg_name))
+                    else:
+                        arg_name = ""
+                        kw_name_1.append("{0}.{1}({2})".format(class_list, kw_name, arg_name))
+                doc_string_value.append(string .format(kw_name, driver, arg_name))
+            else:
+                for argument in argument_list:
+                    argument_name = argument.get('name')
+                    if argument_name is not None:
+                        arg_value = argument.get('value')
+                        if arg_value is None or arg_value is False:
+                            arg_value = argument.text
+                        arg_list = argument_name + "=" + "\'" + arg_value + "\',"
+                        arg_name_list += arg_list
+                kw_name_1.append("{0}.{1}({2})".format(class_list, kw_name, arg_name_list))
+                doc_string_value.append(string .format(kw_name, driver, arg_name_list))
+                arg_name_list = ""
+    return doc_string_value, subkw_list, object_list_new, import_action_list, kw_name_1
+
+def get_relative_path_for_kw_seq_temp():
+    """ To get the absolute path of keyword sequencing template"""
+    katana_exe = sys.argv[0]
+    katana_py_file = katana_exe.split('/')
+    if len(katana_py_file) > 1:
+        katana_py_file = katana_py_file[:-1]
+        katana_py_file = '/'.join(katana_py_file)
+        template_dir = os.getcwd() + '/' + katana_py_file + '/kw_seq_template'
+        template_dir_1 = os.getcwd() + '/' + katana_py_file + '/kw_seq_temp'
+    else:
+        template_dir = 'kw_seq_template'
+        template_dir_1 = 'kw_seq_temp'
+    return template_dir, template_dir_1
+
+
+def get_method_name(line):
+    """ To check whether the user provided wrapper keyword name already exists in user provided
+        action file from keyword sequencing tool UI screen"""
+    if line.strip().startswith("def "):
```

Comment:
this condition will pass if a docstring line starts with 'def ' which is not correct.